rmk/src/debounce.rs

```diff
@@ -48,37 +48,34 @@ impl<const INPUT_PIN_NUM: usize, const OUTPUT_PIN_NUM: usize>
     }
 
     /// Per-key debounce, same with zmk's debounce algorithm
-    pub(crate) fn debounce(
+    pub(crate) fn has_changed(
```

Comment:
`self.debouncer.has_changed` is a little bit confusing for me, it feels like this function would return whether the *debouncer's state* has changed, not the *key state*. Changing it to `self.debouncer.detect_change_with_debounce` or something similar would be better imo.

Reply:
`detect_change_with_debounce` then.